Opti2 to master

Juju49 opened the pull request:
branche fonctionnelle indépendamment à partir du commit 
[4ebcdfe](https://github.com/are00dynamic-2018/Souverains-des-flocons/commit/4ebcdfe8e6251cb39d53aae063d1fd2574c72645). Il serait peut-être temps de pull.

Commit:
threading attention: non opti!!

Juju49 committed:
quelques optimisations

optis graphiques avec pas de redraw des cellules identiques
introduction du threading pour faire une boucle de dessin
+ probleme: pas de redraw au cours de "auto" pour l'instant

Juju49 committed:
Flocon express

temps de génération /3

Juju49 committed:
Fonctionnel

grille agrandie à r=41
temps de traitement 0.2s par step
affichage corrigé: plus d'overlap de texte de slider

Mistergix commented:
Salut Julian, t'as fait du taf de ouf !

Y'a juste quelques petits trucs avec lesquels je ne suis pas d'accord (au niveau du model), ça te dit qu'on regarde ça ensemble jeudi ?

Dans tous les cas t'as géré !

Juju49 committed:
ça marche encore mieux

corrections demandées par Nico ce mercredi:
somme/12
Reset qui marche

à faire:
implem de alpha

Commit:
update model

Commit:
modifs d'interface

des fix à droite à gauche

Commit:
view update

Mistergix committed:
Merge branch 'master' into opti2
